src/wallet/components/ConnectWallet.tsx

```diff
@@ -58,6 +63,14 @@ export function ConnectWallet({
     setIsOpen(!isOpen);
   }, [isOpen, setIsOpen]);
 
+  // Effects
+  React.useEffect(() => {
```

Comment:
it looks like the convention is to import hooks directly. `React.____` seems to be used for types, but not for hooks.

Reply:
Thanks! Fixed